pkgs/by-name/gi/gitlab-runner/package.nix

```diff
@@ -50,13 +55,26 @@ buildGoModule rec {
     rm executors/docker/services_test.go
   '';
 
+  postPatch = ''
+    substituteInPlace Makefile --replace-fail "export VERSION := \$(shell ./ci/version)" "export VERSION := ${version}"
+    substituteInPlace Makefile --replace-fail "REVISION := \$(shell git rev-parse --short=8 HEAD || echo unknown)" "REVISION := v${version}"
```

Comment:
Perhaps I am wrong but, I think the `.git` folder is stripped out, so `git rev-parse` won't work.
Have you tested this?

Reply:
That's correct. I switched back to draft, as I made a mistake while pushing. 17.9.2 introduced further changes (mroe tests fail when not there is no .git directory). I'm currently trying some more things...